GitDiffMargin/Properties/AssemblyInfo.cs

```diff
@@ -32,3 +33,17 @@
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("3.2.0.0")]
 [assembly: AssemblyFileVersion("3.2.0.0")]
+
+#if true // Use this block if Visual Studio 2010 still needs to be supported
+[assembly: Guid("c279cada-3e46-4971-9355-50f43053d1b3")]
+[assembly: ProvideBindingPath]

```

Comment:
@laurentkempe I find it interesting that we've never had problems with Git Diff Margin failing to load the extra assemblies previously included in the VSIX. I guess it's possible that some of those were provided by other extensions and/or Visual Studio itself in some way.